src/reduced_lung/src/4C_reduced_lung_aaa_approx.hpp

```diff
@@ -0,0 +1,131 @@
+// This file is part of 4C multiphysics licensed under the
+// GNU Lesser General Public License v3.0 or later.
+//
+// See the LICENSE.md file in the top-level for license information.
+//
+// SPDX-License-Identifier: LGPL-3.0-or-later
+
+#ifndef FOUR_C_REDUCED_LUNG_AAA_APPROX_HPP
+#define FOUR_C_REDUCED_LUNG_AAA_APPROX_HPP
+
+#include "4C_config.hpp"
+
+#include "4C_linalg_serialdensematrix.hpp"
+
+#include <functional>
+
+FOUR_C_NAMESPACE_OPEN
+
+namespace ReducedLung
+{
+  /**
+   * @brief Options for the AAA (Adaptive Antoulas-Anderson) rational approximation.
+   *
+   * Controls the stopping tolerance and the maximum number of support points
+   * used by the greedy AAA iteration.
```

Comment:
```suggestion
```

Just document the options inside. You will for sure forget to update this header; at least I always do ;)